src/faq.md

```diff
@@ -191,7 +191,7 @@ inputs:
 <a name="enuminputs"></a>
 ## Enum Inputs ⚜️
 
-For command-line flags that require a specific input as the argument, an enum type can be declared in CWL. **Specifying null here is known as long form style. It does the same thing as the question mark on the other inputs.**
+For commandline flags that require a specific input as the argument an enum type can be declared in CWL. **Specifying null here is known as long form style. It does the same thing as the question mark on the other inputs.**
```

Comment:
In other parts of the user guide we have command-line, sometimes command line. But not commandline, so better stick with one of these two (probably command-line) I reckon.